lib/src/Uncertainty/Algorithm/Analytical/AnalyticalResult.cxx

```diff
@@ -233,15 +234,16 @@ void AnalyticalResult::computeMeanPointInStandardEventDomain() const
   Scalar scaling = hasoferReliabilityIndex_ * p_standardDistribution->computeRadialDistributionCDF(hasoferReliabilityIndex_, true);
   Scalar a = hasoferReliabilityIndex_;
   // Quadrature rule
-  Point weights;
-  const Point nodes(p_standardDistribution->getGaussNodesAndWeights(weights));
-  const UnsignedInteger nodesSize = nodes.getSize();
+  const UnsignedInteger integrationNodesNumber = ResourceMap::GetAsUnsignedInteger("Distribution-DefaultIntegrationNodesNumber");
```

Comment:
@jschueller for these distributions, we use other keys (Student-MarginalIntegrationNodesNumber",  "MaximumEntropyOrderStatisticsDistribution-CDFIntegrationNodesNumber", ...)

@regislebrun ok for internal use but the specific AnalyticalResult should have it is own key